contrib/razor/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "razor"
+pkgver = "2.86"
+pkgrel = 0
+build_style = "perl_module"
+hostmakedepends = [
+    "gmake",
+    "perl",
+]
+makedepends = ["perl"]
+depends = ["perl", "perl-uri"]
+pkgdesc = "Distributed, collaborative spam detection and filtering network"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "Artistic-2.0"
+url = "https://razor.sourceforge.net/index.html"
+source = f"https://cpan.metacpan.org/authors/id/T/TO/TODDR/Razor2-Client-Agent-{pkgver}.tar.gz"
```

Comment:
this should use CPAN_SITE like other perl modules

Reply:
okay